$NetBSD$

* no matter libpinyin tools found or not, we can build it
* install missing files

--- data/CMakeLists.txt.orig	2013-07-15 20:34:05.000000000 +0000
+++ data/CMakeLists.txt
@@ -4,6 +4,9 @@ foreach(icon pinyin-libpinyin shuangpin-
     install(FILES ${icon}.png
             DESTINATION share/fcitx/imicon)
 endforeach()
+
+if (LIBPINYIN_TOOLS_FOUND)
+
 set(_LIBPINYIN_TABLE art gb_char history nature scitech
                      culture gbk_char life society economy geology merged
                      sport)
@@ -17,7 +20,8 @@ endforeach()
 set(ZHUYIN_DATA_FILE_NAME model.text.20130308.tar.gz)
 set(ZHUYIN_URL "http://download.fcitx-im.org/data/${ZHUYIN_DATA_FILE_NAME}")
 set(ZHUYIN_ORGDATA ${_LIBPINYIN_TABLE_FILES} interpolation2.text)
-set(ZHUYIN_DATA bigram.db ${_LIBPINYIN_BIN_FILES}  phrase_index.bin  pinyin_index.bin)
+set(ZHUYIN_DATA bigram.db ${_LIBPINYIN_BIN_FILES}  phrase_index.bin
+                pinyin_index.bin addon_phrase_index.bin addon_pinyin_index.bin)
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/table.conf ${CMAKE_CURRENT_BINARY_DIR}/table.conf COPYONLY)
 
@@ -41,5 +45,9 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR
               ${_LIBPINYIN_GEN_BIN_FILES}
               ${CMAKE_CURRENT_BINARY_DIR}/phrase_index.bin
               ${CMAKE_CURRENT_BINARY_DIR}/pinyin_index.bin
+              ${CMAKE_CURRENT_BINARY_DIR}/addon_phrase_index.bin
+              ${CMAKE_CURRENT_BINARY_DIR}/addon_pinyin_index.bin
               table.conf
         DESTINATION ${FCITX_LIBPINYIN_ZHUYIN_DATADIR})
+
+endif()
